src/js/juju-cards-v1.0.0.js

```diff
@@ -116,6 +122,17 @@ let jujuCards = () => {
       detailsLink = `${ownerLink}/${name}`;
     }
     let addLink = `${demoDomain}/?deploy-target=${id}`;
+    let deployTitle = 'Deploy with Juju';
+
+    if (typeof(card.dataset.dd) !== 'undefined') {
+      addLink = `${demoDomain}/?dd=${id}`;
+      deployTitle = 'Deploy with JAAS';
+    }
+
+    let seriesEle = ``;
+    if (series) {
```

Comment:
Sometimes they're not and you get a floating bullet, this hides the floating bullet.